### PR TITLE
[FormRecognizer] Moving Receipt snippet to end of README

### LINKED ISSUE #12655: Move Receipt snippet to end of README
We originally put the [Recognize Receipts](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer#recognize-receipts) snippet as the first snippet in the README because it was short and simple and easy to get started with.  Now that it's more complex, let's put it between Recognize Custom Forms and Train a Model.

### PR DESCRIPTION
**Pull request by kinelski**
Fixes #12655.

As suggested by the issue description, putting the Receipt snippet between Recognize Custom Forms and Train a Model.

### CODE REVIEW

**Comment by kinelski** [comment]
@annelo-msft Should we be concerned about reordering samples as well (content => custom => receipt)?

**Comment by maririos** [comment]
I would say yes to re-ordering samples too. Also, make sure to communicate this with the other languages so they can make those changes too.

And just to double check, the idea is to merge this PR for Preview.4, right?

File: sdk/formrecognizer/Azure.AI.FormRecognizer/README.md [hunk 1]
```diff
@@ -130,6 +130,58 @@ The following section provides several code snippets illustrating common pattern
 * [Train a Model](#train-a-model)
```
Comment (maririos) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/README.md:130]: we should change the order here too

**Comment by kinelski** [comment]
> And just to double check, the idea is to merge this PR for Preview.4, right?

Yes. It's likely that we'll need to merge an update to our testing region for preview 3 so, if you prefer, we can merge this one later.

**Review by annelo-msft — APPROVED** [review]
Looks good

**Review by maririos — APPROVED** [review]

**Pull request merged**